internal/providers/declaration_info.bzl

```diff
@@ -26,14 +26,16 @@ Note: historically this was a subset of the string-typed "typescript" provider.
     # TODO(alexeagle): The ts_library#deps attribute should require that this provider is attached.
     # TODO: if we ever enable --declarationMap we will have .d.ts.map files too
     fields = {
+        "ambient_declarations": "A depset of typings files for ambient srcs",
```

Comment:
I feel like throughout, we need a bit more explanation of what is an "ambient src".
Is this the same term of art used in someplace else so at least it's searchable?